frontend/app/src/services/assets/asset-api.ts

```diff
@@ -287,4 +287,17 @@ export class AssetApi {
       })
       .then(handleResponse);
   }
+
+  restoreAssetsDatabase(
+    reset: String,
+    ignore_warnings: boolean
```

Comment:
I know I said the snake case transformer is unnecessary, but would it be easy to make this camelcase and use the transformer in data?

Reply:
Ah sorry. The need for this parameter came later and forgot about that. Will change sir (hopefuly)